assets/game_projects/fighter/src/main.py

```diff
@@ -221,18 +226,20 @@ def _process_inputs(self) -> None:
             camera_position = Camera.get_viewport_position()
             camera_position += Vector2.DOWN()
             Camera.set_viewport_position(position=camera_position)
-
-        if Input.is_action_pressed(action_name="camera_shake"):
-            x_intensity_max = 5.0
-            y_intensity_max = 5.0
+        if Input.is_action_just_pressed(action_name="camera_shake"):
+            self.x_intensity_max = self.x_shake
+            self.y_intensity_max = self.y_shake
+        if self.x_intensity_max > 0 and self.y_intensity_max > 0:
             Camera.set_offset(
```

Comment:
Any logic that needs to be done here can be handled by the camera shaker object.